pynxtools/dataconverter/README.md

```diff
@@ -37,9 +37,28 @@ Options:
                                   parameters the converter supports.
   --undocumented                  Shows a log output for all undocumented
                                   fields
+  --merge-partial                 Merges partial NeXus files provided as
+                                  different --input-file to the converter.
+  --mapping TEXT                  Takes a <name>.mapping.json file and
+                                  converts data from given input files.
```

Comment:
Does it *have* to be a `.mapping.json` file or would the reader also take a `.json` file? In the latter case I would just say json file but add a sentence that the convention is `.mapping.json`.

Reply:
It has to be a .mapping.json. A regular json file can be used as a data file. So should I keep it as it is or you recommend a better phrasing?

Reply:
No then the phrasing is fine.